tox.ini

```diff
@@ -38,7 +38,6 @@ setenv =
     # applications behave identically, we also set LANG explicitly.
```

Comment:
Not related to your changes but while you're in here can you you also change `OSBenchmark` to `OpenSearch Benchmark` on line 3. 

I struggled to put this comment on the line I actually wanted...

Reply:
Sure will do once the current run of integ tests finishes (pushing a new commit resets them)